prepare.sh

```diff
@@ -37,11 +37,4 @@ cat ~/.sbuildrc
 echo "Checkout workspace"
 
 mkdir src
-case $REPOS_FILE in
-  http*)
-    curl -sSL "$REPOS_FILE" | vcs import --recursive src
-    ;;
-  *)
-    vcs import --recursive src < "$REPOS_FILE"
-    ;;
-esac
+vcs import --recursive --input  "$REPOS_FILE" src
```

Comment:
can you `git revert b228fee` first, then add this on top and then your other changes?

Reply:
Makes sense. :+1:

Reply:
Sorry, I take that back. [curl is still a dependency that should be installed](https://github.com/jspricke/ros-deb-builder-action/blob/dd94ff4e014a3a33d03db6cf13178ebb55ada964/build.sh#L25)